pkgs/applications/networking/instant-messengers/signal-desktop/default.nix

```diff
@@ -0,0 +1,95 @@
+{ stdenv, lib, fetchurl, dpkg, electron, gnome2, atk, cairo, gdk_pixbuf
+, glib, zulu, zulu8, freetype, fontconfig, fontconfig_210, dbus_daemon
+, xlibs, gnome3, glib-tested, nss, nspr, alsaLib, cups, expat, nodejs
+, libudev
+}:
+let
+  rpath = lib.makeLibraryPath [
+    alsaLib
+    atk
+    cairo
+    cups.lib
+    dbus_daemon.lib
+    electron
+    expat
+    fontconfig
+    fontconfig_210
+    freetype
+    gdk_pixbuf
+    glib
+    glib-tested
+    gnome2.GConf
+    gnome2.gtk
+    gnome2.pango
+    gnome3.gconf
+    libudev
+    nodejs
+    nspr
+    nss
+    stdenv.cc.cc
+    xlibs.libX11
+    xlibs.libX11
+    xlibs.libXScrnSaver
+    xlibs.libXcomposite
+    xlibs.libXcursor
+    xlibs.libXdamage
+    xlibs.libXext
+    xlibs.libXfixes
+    xlibs.libXi
+    xlibs.libXrandr
+    xlibs.libXrender
+    xlibs.libXtst
+    xlibs.libxcb
+    zulu
+    zulu8
+  ] + ":${stdenv.cc.cc.lib}/lib64";
```

Comment:
Quite a few unnecessary imports here, see https://github.com/NixOS/nixpkgs/pull/31337 for a minimal set